doc/docker-compose/import-dataset.yml

```diff
@@ -0,0 +1,65 @@
+version: '3.2'
+
+services:
+  databasedump:
+    image: ${DATASET_DBDUMP_IMAGE}
+    volumes:
+      - type: volume
+        source: dbdump
+        target: /dbdump
+        volume:
+          nocopy: false
+
+  import-db:
+    # Using dev for now, because we need mysql client
+    image: ${PHP_IMAGE_DEV}
+    environment:
+     - DATABASE_USER
+     - DATABASE_PASSWORD
+     - DATABASE_NAME
+     - TERM=dumb
+    networks:
+     - stack-db_db
+    volumes:
+      - type: volume
+        source: dbdump
+        target: /dbdump
+        volume:
+          nocopy: true
+    depends_on:
+     - databasedump
+    command: >
+      /bin/bash -c "
+      time mysql -u $${DATABASE_USER} -p$${DATABASE_PASSWORD} $${DATABASE_NAME} -h db< /dbdump/ezp.sql"
+
+  import-vardir:
+    image: ${DATASET_VARDIRDUMP_IMAGE}
+    volumes:
+     - vardir:/nfs/vardirs
+    environment:
+     - DATASET_VARDIR
```

Comment:
should this rather be `DATASET_VARDIR =${APP_PROD_IMAGE}` to avoid duplicating the variable across two env variables?